.github/workflows/docker.yaml

```diff
@@ -27,6 +27,6 @@ jobs:
         file: Dockerfile
         # push: ${{ startsWith(github.ref, 'refs/tags/') }}
         push: true
-        tags: scrolltech/l2geth:${{github.ref_name}}
+        tags: scrolltech/l2geth:${{github.ref_name}},scrolltech/l2geth:latest
```

Comment:
we should discourage people from using the `latest` tag out of ambiguity reasons and to avoid accidental upgrades. but at least we should be clear about what it means. 

- if it is _any_ pushed tag, this will likely cause trouble for users that used the latest tag and happen to pull an updated image on a production node
- imo we should play save and only update the latest tag for stable releases for mainnet

Reply:
isn't it that if we push another newer tag, even it's not using `latest`, it will still overwrite `latest`?

Reply:
Yes, I was also wary about this approach. My thinking was that maybe we could only run this for tags on the `develop` branch. But now I prefer Jonas's proposal, we can build and push the `latest` tag for stable releases (or by manually triggering a GitHub action).

Reply:
changed it to push latest tag on published releases